scarlet/source.py

```diff
@@ -189,9 +190,23 @@ def __init__(
         # retain center as attribute
         self.center = morphology.center
 
-    def init_morph(self, sky_coord, frame):
+    @staticmethod
+    def init_morph(frame, sky_coord, boxsize=None):
```

Comment:
I get why you made this a static message, but is there a reason that you changed the order of the arguments?

Reply:
same order as the constructor